src/components/table/Table.vue

```diff
@@ -125,14 +125,18 @@
                 default: 20
             },
             paginationSimple: Boolean,
-            html: Boolean
+            html: Boolean,
+            checkedRows: {
+                type: Array,
+                default: () => []
+            }
```

Comment:
Using just the [] or {} as default value on props results in warnings being thrown by Vue, this is the solution as described by Vue's creator: https://github.com/vuejs/vue/issues/1032